test/unittest/ndarray.jl

```diff
@@ -298,6 +298,15 @@ function test_eltype()
   end
 end
 
+function test_kwargs()
+  info("NDArray::kwargs")
+  dims1 = (2,3,4)
+
+  x = mx.empty(dims1)
+  tx = mx.transpose(x, axes=(2,1,3))
+  @test size(tx) == (3,2,4)
```

Comment:
Could you actually generate a random matrix, and use Julia's build-in swap axis function to compare against the array contents of the libmxnet transposed version?